src/main/java/frc/robot/subsystems/SuperStructure.java

```diff
@@ -393,6 +393,20 @@ public Command algaeLevelTwoThreeIntake() { // removes algae from lower reef
         .withName("Algae Ground Intake");
   }
 
+  public Command algaeLevelThreeFourIntakeWithoutElevator() { // removes algae from upper reef
```

Comment:
These two commands seem identical except for the logging.  Why are two different commands needed here?  If you need different logging, make it a single command, but pass it a parameter to define what "level" it's at and use that for logging.